CalendarFXView/src/main/java/com/calendarfx/model/EntryEditAction.java

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright (C) 2017 Dirk Lemmermann Software & Consulting (dlsc.com)
+ * <p>
+ * This file is part of CalendarFX.
+ */
+package com.calendarfx.model;
+
+/**
+ * class used for parameter of {@link com.calendarfx.view.DateControl#entryActionPolicy} functional interface
+ */
+public class EntryEditAction {
+
+    /**
+     * the entity the operation is operated on
+     */
+    private final Entry<?> entry;
+
+    /**
+     * the operation
+     */
+    private final EditOperation editOperation;
+
+    public EntryEditAction(Entry<?> entry, EditOperation editOperation) {
```

Comment:
Also include a reference to the DateControl where the policy will be invoked. This closes the gap between model and view.